packages/scratch-gui/src/components/connection-modal/connection-modal.jsx

```diff
@@ -24,8 +24,23 @@ const PHASES = keyMirror({
     updatePeripheral: null
 });
 
-const ConnectionModalComponent = props => (
-    <Modal
+const ConnectionModalComponent = props => {
```

Comment:
@MarshTheTester, we'll need to specifically check hardware extensions for the next Android release. We should include both new (CDM) and old (non-CDM) tablets.